### ISSUE TITLE
fe follower restart wait after alter enable_persistent_index

### ISSUE THREAD

**Issue by chaiyx** [StarRocks/starrocks#45207]
### Steps to reproduce the behavior (Required)
1. Use kubernetes deploys FE
2. Start to run SR In shared-data mode
3. Create table with Properties("enable_persistent_index"="false")
4. ALTER TABLE xxx SET ("enable_persistent_index"="true")
5. Restart fe follower

### Expected behavior (Required)
fe follower start successfully

### Real behavior (Required)
fe follower warn log:
```
 [GlobalStateMgr.waitForReady():1260] It took too much time for FE to transfer to a stable state(LEADER/FOLLOWER), it maybe caused by one of the following reasons: 1. There are too many BDB logs to replay, because of previous failure of checkpoint(you can check the create time of image file under meta/image dir). 2. Majority voting members(LEADER or FOLLOWER) of the FE cluster haven't started completely. 3. FE node has multiple IPs, you should configure the priority_networks in fe.conf to match the ip record in meta/image/ROLE. And we don't support change the ip of FE node. Ignore this reason if you are using FQDN. 4. The time deviation between FE nodes is greater than 5s, please use ntp or other tools to keep clock synchronized. 5. The configuration of edit_log_port has changed, please reset to the original value. 6. The replayer thread may get stuck, please use jstack to find the details.
```

### StarRocks version (Required)
 - Now is 3.2.6, upgrade from 3.2.2

**Comment by sduzh** [comment]
Should have been fixed in #44597

**Comment by chaiyx** [comment]
Is there anyone following up on this issue and pr?
